*Note:
  I've cloned repo from source on Win directory to Linux local dir. Then I tried to add remote, but in outptu it tells me
  that such remote (origin) already exists. "OK, then try to push commits" - I thought. (If someone knows why does is it,
  please tell me
  ###
  Here
  ###
  I've created repo in GitHub, but didn't add remote in Win source)
  And when I tried to push commits it showed me next:
  ---
    remote: error: refusing to update checked out branch: refs/heads/master
    remote: error: By default, updating the current branch in a non-bare repository
    remote: error: is denied, because it will make the index and work tree inconsistent
    remote: error: with what you pushed, and will require 'git reset --hard' to match
    remote: error: the work tree to HEAD.
    remote: error: 
    remote: error: You can set 'receive.denyCurrentBranch' configuration variable to
    remote: error: 'ignore' or 'warn' in the remote repository to allow pushing into
    remote: error: its current branch; however, this is not recommended unless you
    remote: error: arranged to update its work tree to match what you pushed in some
    remote: error: other way.
    remote: error: 
    remote: error: To squelch this message and still keep the default behaviour, set
    remote: error: 'receive.denyCurrentBranch' configuration variable to 'refuse'.
  ---
  It looks like that I didn't synchronized Linux repo with Windows.

